packages/frontend/src/components/wallet-migration/modals/CleanKeysModal/AccessKeyList.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import classNames from '../../../../utils/classNames';
+import Accordion from '../../../common/Accordion';
+import Checkbox from '../../../common/Checkbox';
+import ChevronIcon from '../../../svg/ChevronIcon';
+import { ButtonsContainer, StyledButton } from '../../CommonComponents';
+
+const AccessKeyListContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+
+    .access-key {
+        display: flex;
+        flex: 1;
+        font-weight: 600;
+        padding: 16px 0;
+        border-bottom: 1px solid lightgray;
+
+        .public-key {
+            flex: 7;
+
+            .key-prefix {
+                font-family: monospace;
+            }
+        }
+
+        .remove-checkbox {
+            flex: 1;
+        }
+    }
+
+    .account {
+        background-color: #fafafa;
+        border-bottom: 1px solid #f0f0f0;
+        display: flex;
+        flex: 1;
+        flex-direction: row;
+        width: 100%;
+
+        .account-detail {
+            flex: 7;
+
+            .account-id {
+                font-weight: bold;
+                text-overflow: ellipsis;
+            }
+
+            .keys-to-remove {
+                color: #de2e32;
+            }
+        }
+
+        .expand-keys {
+            flex: 1;
+            padding: 16px 0 0 0;
+        }
+    }
+`;
+
+const AccessKeyList = ({ account, onClose, onNext, selectKey, selectedKeys }) => {
+    const [expanded, setExpanded] = useState(true);
+    return (
+        <AccessKeyListContainer>
```

Comment:
we should use `Container` component from `CommonComponents` instead
(it has common style props, eg `title` class name is applied on h3 but it is not applied properly because it's expecting `Container` component)